src/blog/multiPage.html

```diff
@@ -95,7 +95,7 @@
               </div>
             </div> <!-- end col -->
 
-            <div class="menu-socials hidden-sm hidden-xs">
+            <div class="menu-socials hidden-sm hidden-xs" style="display:none;">
```

Comment:
Do you think we should comment these blocks out or remove them from the codebase altogether?